### PR TITLE
enable auth + various fixes and format

### PR DESCRIPTION
**Pull request by echarles**
Enable auth + various fixes and format.

The next PR will ensure proper navigation after the form submit.

### CODE REVIEW

File: ksmm/_version.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    for settings in HERE.rglob("package.json"): 
+        try:
+            with settings.open() as f:
+                return json.load(f)["version"]
+        except FileNotFoundError:
+            pass
+
+    raise FileNotFoundError(f"Could not find package.json under dir {HERE!s}")
+
+__version__ = _fetchVersion()
```
Comment (Carreau) [inline ksmm/_version.py:18]: +1

**Comment by Carreau** [comment]
LGTM !

**Comment by Carreau** [comment]
And thanks, good work I'll try this week-end.

**Comment by echarles** [comment]
Thx @Carreau

**Pull request merged**